.github/workflows/builder_image_golang.yaml

```diff
@@ -2,9 +2,17 @@ name: builder_image_golang
 on:
   push:
     branches: [develop]
-    paths: [images/builder-golang/**, .github/workflows/image-builder-golang.yml]
+    paths:
+      [
+        distrib/images/builder-golang/**,
+        .github/workflows/image-builder-golang.yml,
+      ]
   pull_request:
-    paths: [images/builder-golang/**, .github/workflows/image-builder-golang.yml]
+    paths:
+      [
+        distrib/images/builder-golang/**,
+        .github/workflows/image-builder-golang.yml,
+      ]
```

Comment:
Formatter / linter?

Reply:
No, long(er) lines triggered the format change.

Reply:
The default vscode one that is Redhat's yaml formatter.